src/transaction/log_replication.cpp

```diff
@@ -519,18 +519,10 @@ namespace cublog
     return m_redo_lsa;
   }
 
-  log_lsa replicator::get_lowest_unapplied_lsa () const
+  log_lsa
+  replicator::get_lowest_unapplied_lsa () const
```

Comment:
I think this function should not be called because, on PTS it will be superseeded by the derived one.
But, for now, just add a TODO to replace the function with an assert.
Am I correct?

Reply:
Yes, I will add an assert